README.md

```diff
@@ -29,6 +29,7 @@ Deploys the [`cas-provision` helm chart] to every namespace used by the team. Th
 - a DockerHub registry credential
 - a `SysdigTeam` object, which is a custom resource created by platform services to grant access to the Sysdig monitoring platform.
 - various secrets containing credentials used by our applications
+- Utilizing `gcp` (the Google Cloud Platform CLI), creates buckets for TF state for every namespace used by the team. Relies on a being authorized with a service account (credentials stored in the team's password manager) with storage permissions on the project.
```

Comment:
Should note that gcloud CLI errors when a bucket already exists created by the same owner.